devel/ddev/Portfile

```diff
@@ -0,0 +1,335 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/ddev/ddev 1.23.5 v
+
+checksums           rmd160  ed4bb63c2d40888fa850e92f956b923570d4f0d5 \
```

Comment:
Since `ddev` needs a git checkout, you can switch MacPorts to fetching using `git` instead of downloading the archive.

Use `fetch.type git`.  You can then remove the `checksums` since you won't be using a tarball anymore.

Example here: https://github.com/macports/macports-ports/blob/master/security/grype/Portfile#L24